packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1555,15 +1555,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         this.verifyNotClosed();
 
         let clientSequenceNumber: number = -1;
+        const opMetadataInternal = { ...opMetadata };
```

Comment:
Compare to the code before, there is an extra object copy here when `!(this.flushMode === FlushMode.Manual && !this.needsFlush)`
We do use `let` instead of `const` here and only do the object copy in the `if` below when we need to add `batch: true`.